R/tab_create_modify.R

```diff
@@ -214,11 +218,14 @@ tab_spanner <- function(data,
 tab_spanner_delim <- function(data,
                               delim,
                               columns = everything(),
+                              split = c("last", "first"),
```

Comment:
Introduce new params as the last param, unless there's a good reason not to--so it's not a breaking change

Reply:
Changed now, thanks!